docs/setup-dev-environment.md

```diff
@@ -57,129 +130,282 @@ EOF
 
 Then run source to make sure node and npm are on your path:
 
-```
+```bash
 source ~/.profile
 ```
 
 You can check the updated variables as follows
 
-```
+```bash
 echo $PATH
 echo $NODEJS_HOME
 ```
 
-## Configuring Postgres
+### :large_blue_circle: Configuring PostgreSQL
 
 Now we configure postgres. First ensure postgres is running.
 
-`service postgresql start`
+```bash
+service postgresql start
+```
 
 Ensure the `en_US` locale exists.
 
-`sudo locale-gen en_US.UTF-8`
+```bash
+sudo locale-gen en_US.UTF-8
+```
 
 Configure the database to listen on network connection.
 
-`sudo sed -i "s/#\?listen_address.*/listen_addresses '*'/" /etc/postgresql/10/main/postgresql.conf`
+```bash
+sudo sed -i "s/#\?listen_address.*/listen_addresses '*'/" /etc/postgresql/12/main/postgresql.conf
+```
 
 Allow authenticated connections from any IP (so includes the host).
 
-`echo "host    all             all             all                     md5" | sudo tee --append /etc/postgresql/10/main/pg_hba.conf > /dev/null`
+```bash
+echo "host    all             all             all                     md5" | sudo tee --append /etc/postgresql/12/main/pg_hba.conf > /dev/null
+```
 
 Restart postgres to pick up config changes.
 
-`service postgresql restart`
+```bash
+service postgresql restart
+```
+
+<!-- Change the password encryption to md5. You will need to edit the postgres config file manually, for example you could use nano, then search for `password_encryption` and change it from `scram-sha-256` to `md5` (making sure you un-comment this line).
```

Comment:
is it intentional that it is added but as commented out?

Reply:
Thanks for spotting this, I can remove, it was a thing I was trying to make postgres-14 work with mapnik.